PaolaSantolloV/react-apprenticeship-capstone3
# Challenge-3

## Description
This PR contain the capstone project 2

- Add husky, prettier and eslint on the project
- Add hooks for the request to firebase
- Add the functional components
- Add test for responsive
- Add test for hooks and functional components
- Add styles on the components
- Add routing 
- Add authentication


## Type of Change




## How should this be tested?
In your terminal run `npm install` after that` npm run start`, now you can see the page on [localhost:3000](localhost:3000)
To run the test` npm run test`, if you want to watch the coverage` npm run test a -- --coverage`.

## What is the deliverable?
Final deliverable

## Screenshots and links

netlify link: [https://jovial-austin-c43723.netlify.app/login](https://jovial-austin-c43723.netlify.app/login)

## Notes
You can access to notes with the following data
```
 email: paola@gmail.com
 password: 1234567890
```
or registering with your own email.

## Checklist
- [ x] I added tests to prove that my fix is effective or my deliverable works.
